ampd/src/handlers/evm_verify_msg.rs

```diff
@@ -371,6 +370,179 @@ mod tests {
         goldie::assert_debug!(event);
     }
 
+    #[test]
+    fn test_voting_verifier_event_vs_legacy_event_parsing() {
```

Comment:
@cgorenflo @cjcobb23 

Upon further inspection of the code I had a feeling that ampd would already support parsing both event versions into the same `PollStartedEvent` value.

Can you guys please sanity check this and ensure the test properly tests for this behavior?